dt-impact-tracer/src/edu/washington/cs/dt/impact/Main/EnhancedResultsFigureGenerator.java

```diff
@@ -162,15 +193,19 @@ private static String generate19(String projectName, double[] orig_values, doubl
             fig19GeoData.add(new GeometricMeanData(getK(i), unenhancedPara, Constants.TD_SETTING.GIVEN_TD,
                     Constants.ORDER.ORIGINAL));
         }
+        // i represents unenhanced, i + 1 represents enhanced
         for (int i = 0; i + 2 <= time_values.length; i += 2) {
-            enhancedParaSpeedup = (time_values[i] / orig_time_value);
+            // enhanced
+            enhancedParaSpeedup = (time_values[i + 1] / orig_time_value);
             if (!nonZeroNumOfDTS[i]) {
-                unenhancedPara = (time_values[i + 1] / orig_time_value);
+                // unenhanced
+                unenhancedPara = (time_values[i] / orig_time_value);
             } else {
-                unenhancedPara = (orig_apfd_value / orig_time_value);
+                // orig
+                unenhancedPara = (orig_time_value / orig_time_value);
             }
+            // enhanced - unenhanced or enhanced - orig if DTS != 0
```

Comment:
Your comment here is inconsistent with what you are actually doing below (what you're doing below is correct, this comment is not).